config.yml

```diff
@@ -8,6 +8,10 @@ defaults:
     publishMetaData:              !env:bool PUBLISH_METADATA
     # Email address to blacklist, useful to mitigate temporary misuse
     emailBlacklist:               !env:json EMAIL_BLACKLIST
+    # ignore tasks with reasonResolved matching something in this list
+    ignoreTaskReasonResolved:
+        - cancelled
```

Comment:
This is spelled with one `l` in taskcluster apparently!

Reply:
Which is apparently the American way, so I guess we can't blame @jonasfj :)

Reply:
Awesome catch BTW, your review skills are legion :)

Reply:
I only noticed because I had just reviewed a jonas patch where it was spelled `canceled` and I thought, "that's a weird spelling". I was going to tell him he was wrong until I checked the docs :p

Reply:
Which is weird because we have two americans here who spell it the wrong way apparently.